packages/@okta/vuepress-site/docs/concepts/iam-overview-identity-management-factors/index.md

```diff
@@ -0,0 +1,246 @@
+
+- Add any number of directories or domains.
+
+- Store, manage, and apply business and authentication policies that control how you handle different sets of users, including customer and partner users.
+
+- Set consistent user access policies with a central policy engine, create policies based on unique user contexts, and employ any combination of MFA factors.
+
+- Maintain user device information in device profiles that you associate with user profiles.
+
+- Define, store, and administer user roles, permissions, policies, and other data.
+
+- Organize users in hierarchical groups, and assign roles, permissions, and policies to them that are inherited by their members.
+
+- Use its out-of-the-box connections with HR systems like Workday, SaaS apps like G Suite, CSV files, and third-party identity providers.
+
+- Get a real-time syslog to troubleshoot and address security issues and use pre-built reports to see how end users use apps and services. You can download or export your data to CSV files, sync it with Security Information and Event Management systems, or access it through its API.
+
+- Admins can view and manage policies, reset passwords, and view and manage profiles of all users in their domains using its API and admin user interfaces.
+
+- Obsolete data is disabled without being deleted and can be provided to authorized admins.
+
+- You can replace your on-premise LDAP and Active Directory.
+
+Learn more:
+
+- [Universal Directory](https://www.okta.com/products/universal-directory/) lists major features and benefits of Universal Directory.
+
+### Profiles and progressive profiling
+
+Okta stores user identity information in user profiles that authorized users and admins can view, manage, and update. You can use progressive profiling to update profile data after users register.
+
+#### Okta user profiles
+
+A typical user profile contains attributes such as a user's first name, last name, username, and email address. A user is any person who is authorized to access your applications, and can be an employee, customer, partner, or other entity.
+
+Okta supports four types of user profiles:
+
+- **Okta user profiles** contain information about individual end users. They have thirty-one default base attributes, and you can add custom attributes.
+
+- **Okta group profiles** contain information about specific user groups. They have base attributes for name and description, and you can add custom attributes.
+
+- **Okta app user profiles** define the attributes that can be pushed to or imported from particular apps in the Universal Directory. It has both base and custom attributes.
+
+- **Okta custom user profiles** are based on Okta user profiles and are used for different types of users, such as administrators, contractors, help desk, and so on. (You can have up to ten user types, with a separate custom user profile for each.) Custom user profiles also have thirty-one default base attributes, and you can add custom attributes.
```

Comment:
Perhaps not mention "thirty-one"? We may add more, then you have to remember to come in and update this....
```suggestion
- **Okta custom user profiles** are based on Okta user profiles and are used for different types of users, such as administrators, contractors, help desk, and so on. (You can have up to ten user types, with a separate custom user profile for each.) Custom user profiles also have thirty-one default base attributes, and you can add custom attributes.
```